packages/core-react/src/walletData/getConnectors.ts

```diff
@@ -0,0 +1,14 @@
+import { WalletsListEthereum } from '@reef-knot/wallets-list';
+
+export const getConnectors = ({ rpc }: { rpc: Record<number, string> }) => {
+  const walletAdapters = Object.values(WalletsListEthereum);
+  const walletDataList = walletAdapters.map((walletAdapter) =>
+    walletAdapter({ rpc }),
+  );
+
+  const connectors = [...walletDataList].map(
+    (walletData) => walletData.connector,
+  );
+
+  return connectors;
```

Comment:
```suggestion
  return walletDataList.map(({connector}) => connector);
```
1. it is already an array, no reason to spread
2. why introducing var to return it immediately?
3. this is opinionated: I think introducing the `walletData` variable is redundant here, destruction can work too (but I totally understand this syntax for shortening the code)

Reply:
Agree, it looks this way because there was some other walletconnect related code. I removed it, but forgot to optimize the code.